index.html

```diff
@@ -42,25 +36,34 @@
     <body>
         <script type="text/javascript">
             $(document).ready(function () {
-                floe.setupUIO({
-                    templatePrefix: "lib/infusion/framework/preferences/html/",
-                    messagePrefix: "lib/infusion/framework/preferences/messages/",
-                    tocTemplate: "lib/infusion/components/tableOfContents/html/TableOfContents.html",
-                    template: "lib/infusion/framework/preferences/html/SeparatedPanelPrefsEditor.html"
-                });
+                floe.setupUIO();
             });
         </script>
 
         <!-- BEGIN markup for Preference Editor -->
         <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the mobile view.
+                A separate panel bar for mobile displays is needed to preserver the correct tab order.
```

Comment:
SP: "preserver" -> "preserve" (possibly copy-pasted from elsewhere, repeated in some other comments)